tools/mcetool.c

```diff
@@ -5719,6 +5747,21 @@ static const mce_opt_t options[] =
                         "set the delay for dumping core from unresponsive lipstick\n"
         },
         {
+                .name        = "set-input-policy-mode",
+                .with_arg    = xmce_set_input_policy_mode,
+                .values      = "enabled|disabled",
+                .usage       =
+                        "allow/deny grabbing of input devices based on input policy\n"
+                        "\n"
+                        "Normally this should be always set to 'enabled'.\n"
```

Comment:
When display is powered up, the touch input grab is kept until mce thinks
a) the display is no longer touched by finger or palm (jolla has palm detection)
b) proximity sensor is not covered (optional)
c) lid sensor says the display is not covered (nexus5 etc)

In practice would:
* make stray input during in-call blanking more possible when proximity sensor is briefly uncovered while the device is held at ear.
* break the "lpm" mode (where only mce is supposed to handle touch events)